Ports/julius/package.sh

```diff
@@ -13,11 +13,31 @@ depends=(
 configopts=(
     "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
 )
+install_dir='/opt/julius'
+launcher_name='Julius'
+launcher_category='Games'
+launcher_command="${install_dir}/julius"
+icon_file='res/julius_32.png'
 
 configure() {
     run cmake "${configopts[@]}" .
 }
 
 install() {
-    run make "${installopts[@]}" install
+    run_nocd mkdir -p "${SERENITY_INSTALL_ROOT}${install_dir}/"
+    run cp -r julius "${SERENITY_INSTALL_ROOT}${install_dir}/"
+    # This is necessary to allow the user to create save files
+    run_nocd chmod a+w "${SERENITY_INSTALL_ROOT}${install_dir}/"
```

Comment:
I'm not sure how much I like this. In any case, `/opt` isn't even mounted read-write, is it?

Reply:
Yeah, doing it this way definitely isn't great. Unfortunately, the game puts the save files in the same directory as all of the other game data (not even in their own subdirectory) and there is no built-in option to change save file location.

Looking at `/etc/fstab` it would seem that `/` is supposed to be mounted read only, but running `chmod a+w /opt/julius` definitely allows me to write save files into the directory. while I can't write anything if I don't do that.

Further investigation shows that I can also do this:
![readonly_root_directory](https://github.com/SerenityOS/serenity/assets/2817754/c132fec4-726b-400b-8dae-9ae28625bb64)

Is the `ro` flag in `/etc/fstab` merely a suggestion? :confused:

Reply:
On a related note, I've noticed that Ports are split between those that use `/opt` for storing game data and those that use `/usr/local/share/games`. Is there a preference for which is better to use?

Reply:
> Is the ro flag in /etc/fstab merely a suggestion?

It shouldn't be (if other *nix systems are anything to go by). Could I interest you in a quick detour into `Kernel`? :^)

> On a related note, I've noticed that Ports are split between those that use /opt for storing game data and those that use /usr/local/share/games. Is there a preference for which is better to use?

I think we primarily just use whatever that particular port uses as its default. Obviously, both are quite undesirable because neither of them is writable for the average user.

Reply:
I've updated the PR to assume that game data files are installed to `/home/anon/Games/julius` rather than `/opt/julius`. This isn't ideal because the launcher assumes the user is `anon`, but I'm not sure there's another place that the user is guaranteed to be able to write to.